hbt/tasks/sync.py

```diff
@@ -53,7 +53,7 @@ def requires(self):
     def output(self):
         return {
             "overlap": self.target("lfn_overlap.json"),
-            "unique_identifiers": self.target("unique_identifiers.parquet"),
+            "index_variabless": self.target("index_variabless.parquet"),
```

Comment:
```suggestion
            "index_variables": self.target("index_variables.parquet"),
```
Typo that is found several time

Reply:
Side effects from multiline editing^^. Thanks for noticing